### PR TITLE
[ARI] Autoscaler Area: add Oliver Mautschke

### PR DESCRIPTION
**Pull request by OliverMautschke**
Add Oliver Mautschke (works for SAP)

### CODE REVIEW

**Review by KevinJCross — APPROVED** [review]

**Comment by OliverMautschke** [comment]
@Gerg Could you please review this?

**Comment by Gerg** [comment]
Per [RFC-0008](https://github.com/cloudfoundry/community/blob/main/toc/rfc/rfc-0008-role-change-process.md#promotion-to-approver), you need 2 existing approvers to review your application and confirm that you meet the approver requirements as defined in [ROLES.md](https://github.com/cloudfoundry/community/blob/main/toc/ROLES.md#approver).

Once that criteria is met, I'll merge this PR.

File: org/cloudfoundry.yml
Comment (stephanme) [inline org/cloudfoundry.yml:2405]: New approvers should not be added anymore to the deprecated, non-standard github teams. These teams will be deleted very soon by #262.

Adding to the WG charter is sufficient. Org automation will add you to https://github.com/orgs/cloudfoundry/teams/wg-app-runtime-interfaces-autoscaler-approvers. 

Not adding to team 'App Runtime Interfaces WG Autoscaler' also helps also to find remaining hidden usage of this team before deletion (like Concourse authorisation).

**Comment by AP-Hunt** [comment]
@OliverMautschke @silvestre are we able to progress this any?

**Comment by silvestre** [comment]
@AP-Hunt: As specified by https://github.com/cloudfoundry/community/blob/main/toc/ROLES.md#requirements-3 the required 20 PRs/issues submitted/reviewed have not yet been reached:

|Number Issues/PRs|repo|reviewed/authored|link|
| ---  | --- | --- | --- |
| 4 | app-autoscaler-release | authored | https://github.com/cloudfoundry/app-autoscaler-release/pulls?q=author%3AOliverMautschke+is%3Amerged+ |
| 8 | app-autoscaler-release | reviewed | https://github.com/cloudfoundry/app-autoscaler-release/pulls?q=reviewed-by%3AOliverMautschke |
| 2 | app-autoscaler | authored | https://github.com/cloudfoundry/app-autoscaler/pulls?q=author%3AOliverMautschke+is%3Amerged |
| 4 | app-autoscaler | reviewed | https://github.com/cloudfoundry/app-autoscaler/pulls?q=reviewed-by%3AOliverMautschke+ |

AFAICS, in sum @OliverMautschke has reviewed/authored 18 substantial issues/PRs.

**Comment by silvestre** [comment]
I have one PR almost ready for review that I Oliver can review and there's another PR by Oliver still open, so Oliver should be able to satisfy the requirement soon.

**Review by silvestre — APPROVED** [review]
Counting @OliverMautschke's contributions on 06-12-2022:

|Number Issues/PRs|repo|reviewed/authored|link|
| ---  | --- | --- | --- |
| 5 | app-autoscaler-release | authored | https://github.com/cloudfoundry/app-autoscaler-release/pulls?q=author%3AOliverMautschke+is%3Amerged+ |
| 12 | app-autoscaler-release | reviewed | https://github.com/cloudfoundry/app-autoscaler-release/pulls?q=reviewed-by%3AOliverMautschke |
| 2 | app-autoscaler | authored | https://github.com/cloudfoundry/app-autoscaler/pulls?q=author%3AOliverMautschke+is%3Amerged |
| 4 | app-autoscaler | reviewed | https://github.com/cloudfoundry/app-autoscaler/pulls?q=reviewed-by%3AOliverMautschke+ |

In sum @OliverMautschke has reviewed/authored 23 substantial issues/PRs fulfilling the approver requirements as defined in [ROLES.md](https://github.com/cloudfoundry/community/blob/main/toc/ROLES.md#approver).

File: org/cloudfoundry.yml
Comment (silvestre) [reply org/cloudfoundry.yml:2405]: @OliverMautschke Can you please remove the changes to `org/cloudfoundry.yml` from this PR?

**Review by Gerg — APPROVED** [review]
Everything is in order. Congratulations on your promotion to approver! 🎉

**Pull request merged**
